packages/components/src/Workflow/Workflow.tsx

```diff
@@ -29,8 +28,8 @@ const WorkflowComponent = ({
   classNameOverride,
   ...restProps
 }: WorkflowProps): JSX.Element => (
-  <div
-    className={classnames([styles.workflowWrapper, classNameOverride])}
+  <Workflow.Wrapper
+    classNameOverride={classnames(classNameOverride)}
```

Comment:
Ah, I don't think I was clear before.

Either remove the whole line and have `classNameOverride` just be part of `...restProps`

```
  ...restProps
  }: WorkflowProps): JSX.Element => (
    <Workflow.Wrapper {...restProps}>  	
```

Or if you want to keep it destructured, then remove `classnames`

```
  classNameOverride,
  ...restProps
  }: WorkflowProps): JSX.Element => (
    <Workflow.Wrapper
      classNameOverride={classNameOverride}
      {...restProps}
    >  	
```

Reply:
Right, read you loud and clear now! I think I'll opt for the second one, to make it clear if anyone were to look at the src code